### PR TITLE
Website: Human-friendly preamble

### PR DESCRIPTION
**Pull request by Pandapip1**
Turns this:
<details>
  <summary>Click to expand</summary>
  
  
  
  
</details>

Into this:
<details>
  <summary>Click to expand</summary>
  
  
  
  
</details>

Test it out yourself (it looks better in context than in screenshots): https://pandapip1.github.io/EIPs/

### CODE REVIEW

**Comment by xinbenlv** [comment]
One data point: I kind of like the table view slightly better...

**Comment by SamWilsn** [comment]
I'm not opposed generally to changing the layout, but I think the template currently in this PR makes it harder to scan for relevant information.

What about something like this:

Untitled

Ignore the fonts, font sizes, and colors :rofl: 

Notable changes:
 - Always display discussion link.
 - Separate author name from link so that I don't accidentally open my mail client.
 - Use color to highlight / separate the category/status/track.

**Comment by Pandapip1** [comment]
Good idea, but that'll be quite a bit more work. I'll move it to draft in the meantime.

**Comment by SamWilsn** [comment]
The giant megaphone looks a bit unprofessional. How about:

image

Can we make the entire alert into a link to the discussion thread (and avoid writing out the whole URL?)

---

Long author lists wrap poorly:

image

I would suggest something like this:

image

Is it possible to prevent a linebreak between the author's name and the opening parenthesis? Maybe with an `&nbsp;`?

Can we make the `@` part of the link for author github usernames?

---

Unless we're actually locking the discussion threads, I think putting `(Archive)` will just confuse people.

---

There's no `Status` on EIP-5133:

image

**Comment by amyn1234** [comment]
Get a way ,very cruel and dankey ,because you are  my relative ,I don't you
say but early answer !

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by Pandapip1** [comment]
> There's no Status on EIP-5133:

I've removed the status for final EIPs. I don't think it really makes sense to indicate that an EIP is final, as it doesn't really make sense to tell the user "it's okay to use this", as it's implied by the existence of the page (which is why the other badges have to exist).

**Comment by amyn1234** [comment]
Yes ,I know but ,to be my fuck your all family

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by amyn1234** [comment]
Fuck off and my fuck your ass mother

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by amyn1234** [comment]
You know yours father a human cruel and (به مثال خودمون کونی هست )because
all relatives dankey and monkey

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by amyn1234** [comment]
Wait for me .one day

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by amyn1234** [comment]
If you could  not pay attention, tell me

در تاریخ دوشنبه ۲۱ نوامبر ۲۰۲۲،‏ ۲۱:۱۵ Pandapip1 ***@***.***>
نوشت:

**Comment by SamWilsn** [comment]
I worry that the absence of a label might appear like a bug, but not enough to make you change it :stuck_out_tongue_closed_eyes:

File: _layouts/eip.html [hunk 1]
```diff
@@ -2,59 +2,89 @@
...
+  <h1 class="page-heading">
+    {% if page.category == "ERC" %}
+      ERC-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
+    {% elsif page.category != "ERC" %}
+      EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
     {% endif %}
+    <a href="{{site.github.repository_url}}/blob/master/{{page.path}}"><svg role="img" aria-label="Source" class="bi flex-shrink-0 me-2" xmlns="https://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><use xlink:href="#bi-code"/></svg></a>
+  </h1>
+  <h3>{{ page.description | xml_escape }}</h3>
+  <table class="table table-borderless">
```
Comment (SamWilsn) [inline _layouts/eip.html:52]:
```suggestion
  <table class="table table-borderless preamble">
```

File: assets/css/style.scss [hunk 2]
```diff
@@ -34,10 +34,14 @@ main.page-content {
   color: #111;
 }
 
+.table-borderless, .table-borderless *, .table-borderless * * {
+  border-style : hidden !important;
+  background-color: rgba(0,0,0,0) !important;
+}
+
```
Comment (SamWilsn) [inline assets/css/style.scss:41]: Probably not the most elegant way to accomplish this, but something like:

```suggestion

table.preamble > tbody > tr > :not(:first-child) {
  width: 100%;
}

table.preamble > tbody > tr > :first-child {
  white-space: nowrap;
}
```

File: assets/css/style.scss [hunk 3]
```diff
@@ -34,10 +34,14 @@ main.page-content {
   color: #111;
 }
 
+.table-borderless, .table-borderless *, .table-borderless * * {
```
Comment (SamWilsn) [inline assets/css/style.scss:37]: Won't `.table-borderless *` match any element under `.table-borderless` (including the ones matched by `.table-borderless * *`)?

File: assets/css/style.scss [hunk 4]
```diff
@@ -34,10 +34,14 @@ main.page-content {
...
+  border-style : hidden !important;
+  background-color: rgba(0,0,0,0) !important;
```
Comment (SamWilsn) [inline assets/css/style.scss:39]: Why are you using `!important` here? I've always had the vague impression that it shouldn't be necessary in most cases.

File: _includes/head.html [hunk 5]
```diff
@@ -3,8 +3,13 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   {% if page.layout == "eip" %}
-    <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
-    <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% if page.category == "ERC" %}
+      <title>EIP-{{ page.eip }}: {{ page.title | escape }}</title>
+      <meta property="og:title" content="EIP-{{ page.eip }}: {{ page.title | escape }}" />
+    {% elsif page.category != "ERC" %}
```
Comment (SamWilsn) [inline _includes/head.html:9]: There's no `else`? O.o

File: _includes/head.html [hunk 6]
```diff
@@ -3,8 +3,13 @@
...
+      <title>ERC-{{ page.eip }}: {{ page.title | escape }}</title>
```
Comment (SamWilsn) [inline _includes/head.html:10]: /me is still opposed to this change >:(

File: assets/css/style.scss (hunk 2 above)
Comment (Pandapip1) [reply assets/css/style.scss:41]: What does this do?

File: assets/css/style.scss (hunk 3 above)
Comment (Pandapip1) [reply assets/css/style.scss:37]: I'm pretty sure `*` only matches one level deep.

File: assets/css/style.scss (hunk 4 above)
Comment (Pandapip1) [reply assets/css/style.scss:39]: The Jekyll CSS loads after this CSS (no way to change that), so `!important` has to be used to override it.

File: _includes/head.html (hunk 5 above)
Comment (Pandapip1) [reply _includes/head.html:9]: I don't know if there is or not, and I couldn't find any documentation saying there is or is not. If it aint broke, don't fix it.

Comment (SamWilsn) [reply _includes/head.html:9]: https://github.com/ethereum/EIPs/pull/6017/files/40207fe4b72a1cc01c6e580787e769cefd99b548#diff-a88dab7bceb531aea80805e037c9a03ba58430da047369e82ba8a7a3270a3af1R9 :expressionless:

File: assets/css/style.scss (hunk 3 above)
Comment (SamWilsn) [reply assets/css/style.scss:37]: You aren't thinking of `.table-borderless > *`?

**Comment by SamWilsn** [comment]
@SamWilsn's magical code seems to have worked (in Firefox at least)!

File: _includes/head.html (hunk 6 above)
Comment (Pandapip1) [reply _includes/head.html:10]: Wait, hold up. I might have reversed the logic (and introduced a typo)
image
image

**Comment by SamWilsn** [comment]
image

Not sure if you're done with these yet, but the bottom of the images should line up with the text baseline (and not the bottom of the bounding box.) The comment bubble link is an excellent idea though!

**Comment by Pandapip1** [comment]
> the bottom of the images should line up with the text baseline

If you have any idea how to do this, your help would be appreciated. But it really *really* doesn't want to work.

**Comment by SamWilsn** [comment]
Something like:

```css
a {
  height: 1ex;
  display: inline-block;
  vertical-align: baseline;
}

.inline-svg {
  width: 100%;
  height: 100%;
  object-fit: cover;
}
```

Probably want to use a class for the `a` block though.

**Comment by Pandapip1** [comment]
Fixed.

File: _layouts/eip.html [hunk 7]
```diff
@@ -2,59 +2,103 @@
...
-  <h1 class="page-heading">
-    EIP-{{ page.eip | xml_escape }}: {{ page.title | xml_escape }}
-    <a href="{{site.github.repository_url}}/blob/master/{{page.path}}"><svg role="img" aria-label="Source" xmlns="https://www.w3.org/2000/svg" width="14" height="16" viewBox="0 0 14 16"><title>Source</title><path fill-rule="evenodd" d="M9.5 3L8 4.5 11.5 8 8 11.5 9.5 13 14 8 9.5 3zm-5 0L0 8l4.5 5L6 11.5 2.5 8 6 4.5 4.5 3z"/></svg></a>
-  </h1>
-  <h3>{{ page.description | xml_escape }}</h3>
-  <table>
-    <tr><th>Author</th><td>{% include authorlist.html authors=page.author %}</td></tr>
-    {% if page["discussions-to"] != undefined %}
-      <tr><th>Discussions-To</th><td><a href="{{ page["discussions-to"] | uri_escape }}">{{ page["discussions-to"] | xml_escape }}</a></td></tr>
+  <span class="h4">
```
Comment (SamWilsn) [inline _layouts/eip.html:21]: I think this font size is too big now.

File: assets/css/style.scss [hunk 8]
```diff
@@ -2,7 +2,7 @@
 # Only the main Sass file needs front matter (the dashes are enough)
 ---
 
-$content-width: 960px;
+$content-width: 100%;
```
Comment (SamWilsn) [inline assets/css/style.scss:5]: This undoes the fix from #6254 / #6231, right? I believe we wanted a media query for this?

**Review by SamWilsn — COMMENTED** [review]
Last complain of mine is related to #6254

File: _layouts/eip.html (hunk 7 above)
Comment (Pandapip1) [reply _layouts/eip.html:21]:
```suggestion
  <span class="h5">
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**